lesson_13/adrian_filimon/lesson_13.py

```diff
@@ -0,0 +1,64 @@
+from math import sqrt
+
+class PerimeterMixin:
+    def perimeter(self):
+
+        class_name = type(self).__name__
+
+        return f"This is the perimeter for {class_name}: {sum(self.sides)}"
+
+
+class Polygon(PerimeterMixin):
```

Comment:
I think that the requirement was to make PerimeterMixin available only for triangle class so it should be passed as an argument only for the triangle class

Reply:
Oh yes.I can change that so it better fits the requirement,  but I wanted to give all the classes that functionality and practice multiple inheritance.